inst/include/RcppEigenWrap.h

```diff
@@ -81,18 +81,21 @@ namespace Rcpp{
         // for plain dense objects
         template <typename T>
         SEXP eigen_wrap_plain_dense( const T& obj, Rcpp::traits::true_type ) {
+            bool needs_dim = T::ColsAtCompileTime != 1;
+            R_xlen_t m = obj.rows(), n = obj.cols();
+            if (needs_dim && (m > INT_MAX || n > INT_MAX)) {
+                Rcpp::stop("array dimensions cannot exceed INT_MAX");
+            }
```

Comment:
The codecov warning is irritating. I am adding a `.codecov.yml` that worked for us in Rcpp.